__test__/create-or-update-branch.int.test.ts

```diff
@@ -297,11 +297,9 @@ describe('create-or-update-branch tests', () => {
       ADD_PATHS_DEFAULT
     )
     await git.checkout(BRANCH)
-    expect(result.action).toEqual('created')
```

Comment:
The result of this test should be `created`. I'm confused about these changes and not really sure what you are trying to fix.

Reply:
Why should a PR be created on a untracked file change?

I've added the `UNTRACKED_FILE` to the [`.gitignore`](https://github.com/peter-evans/create-pull-request/pull/2098/commits/183ff16ab93e3b5ddb17b940bd7ea968478d3321) file.
This changes the behavior of this test `result.action` to `none` instead of `created`.

If I understand this tests correctly the untracked file should NOT result in a created PR.
Since the ignore-file was not yet correct, the tests were evaluated with an incorrect result.
Now the .gitignore is fixed and the tests need also to be fixed, right?

Reply:
> Why should a PR be created on a untracked file change?

Because that's a feature of this action. You can create changes that are as-yet untracked and the action will raise them in a PR. Without this, the action would only work for modifying existing files and you wouldn't be able to add a new file.

Reply:
hi

Reply:
You're right - I missed that part in the documentation.
But should not be a untracked file be added with `--force`?
This could be added as a additional argument.

Otherwise this will not prevent that unintentional changes will be added.

Reply:
There's a section in the readme about [controlling committed files](https://github.com/peter-evans/create-pull-request#controlling-committed-files). Those options should be enough for users to have control over what untracked files are committed.